src/lib/utils/spatial-index.ts

```
@@ -0,0 +1,266 @@
+    }
+
+    return candidates.filter(
+      (item) =>
+        calculateHaversineDistance(center, item.coordinates) <= radiusMeters
+    );
+  }
+
+  findNearest(
+    center: Coordinates,
+    k = 1,
+    maxRadiusMeters = Infinity
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)